src/news/2016-11-30-DEEP-conference-2016.md

```diff
@@ -0,0 +1,21 @@
+---
+title: DEEP Conference 2016
+filename: DEEP-conference-2016
+date: '2016-11-30'
+---
+
+On October 13-14, 2016 the [DEEP -Discovery conference](http://deep.idrc.ocadu.ca)
```

Comment:
Please add a space after the dash in the link text "Deep - Discovery conference"